code/_onclick/human.dm

```diff
@@ -64,7 +64,7 @@
 
 /mob/living/carbon/human/UnarmedAttack(atom/A, proximity, click_parameters)
 
-	if(lying) //No attacks while laying down
+	if(body_position) //No attacks while laying down
```

Comment:
Might be an idea to use `body_position == LYING_DOWN` for these kind of checks for readability since we're replacing `lying` with `body_position` which adds ambiguity.

Reply:
Yeah, it's just not always obvious if something should be `body_position == LYING_DOWN` or actually `body_position != STANDING_UP` so i kinda gave up at some point, i'll go through again and fix these

Reply:
Yea makes sense lot of files to look at I went crosseyed just reading the diff

Reply:
Updated these + made xeno sprites update on more accurate conditions